wolox-books/src/App.vue

```diff
@@ -1,19 +1,14 @@
 <template>
   <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
     <router-view/>
   </div>
 </template>
 
 <style lang="scss">
 #app {
```

Comment:
The way you should do this is creating an `application.scss` in the `src/scss` folder. Then, import it in here. In that file, remember to add the `wolox-equalizer` and add this `font-family` override to `*` :smile: